webapp/app/templates/eligibility/form_marital_status_input.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'eligibility/form_full_width.html' %}
+{% import "macros.html" as macros %}
+
+{% block additional_form_content %}
+    <div id="marital_status_widowed_note" class="hidden mt-4">
+        <span>{{ _('eligibility.marital_status.widowed_note') }}</span>
+    </div>
+{% endblock %}
+
+{% block optional_js %}
+    {{ super() }}
+    <script>
+        function show_hide_widowed_note() {
+            if ($('input[value="widowed"]').is(':checked')) {
+                $('#marital_status_widowed_note').show();
+            } else {
+                $('#marital_status_widowed_note').hide();
+            }
+        }
+
+        $(`input[type="radio"]`).change(show_hide_widowed_note)
```

Comment:
Why not this?
```suggestion
        $(`input[value="widowed"]`).change(show_hide_widowed_note)
```

Reply:
I also thought that would work. BUT: this way, the text ist not hidden when another radio button is clicked. :)